wp-content/themes/avalanche/inc/recipe-excerpt.php

```diff
@@ -10,20 +10,22 @@
 $recipeExcerpt = get_field( 'excerpt' );
 ?>
 
+<div class="related-item-card">
+	<a href="<?php the_permalink(); ?>">
+		<?php
+		// Image within link; forcing empty alt
+		echo wp_get_attachment_image( get_field( 'gallery_image', $recipeId )['id'], 'square_crop', '', array( 'alt' => '' ) )
+		?>
+	</a>
+
+	<div class="h5 related-heading"><?php the_title(); ?></div>
 
-<a href="<?php the_permalink(); ?>">
 	<?php
-	// Image within link; forcing empty alt
-	echo wp_get_attachment_image( get_field( 'image', $recipeId )['id'], 'square_crop', '', array( 'alt' => '' ) )
+	if ( $recipeExcerpt ) :
+		echo '<div class="related-text"><p>' . wp_kses_post( $recipeExcerpt ) . '</p></div>';
+	endif;
 	?>
-</a>
-
-<div class="h6"><?php the_title(); ?></div>
 
-<?php
-if ( $recipeExcerpt ) :
-	echo '<div class="related-text">' . wp_kses_post( $recipeExcerpt ) . '</div>';
-endif;
-?>
+	<a class="link-white" href="<?php the_permalink(); ?>"><?php esc_html_e( 'VIEW RECIPE', 'avalanche' ); ?></a>
```

Comment:
```suggestion
	<a class="link-white" href="<?php the_permalink(); ?>"><?php esc_html_e( 'VIEW RECIPE', 'avalanche' ); ?></a>
```
I'd maybe slap a strtoupper() in PHP or a text-transform: uppercase in css. That way the translation files aren't always dependent on being uppercase too.